lang/lang_en_es.txt

```diff
@@ -94,10 +106,22 @@
 "Bed level correct"
 "Corr. de la cama"
 
+#MSG_BELTTEST c=17
+"Belt test        "
+"Test cinturon"
```

Comment:
for some reason there are additional spaces in the EN text

Reply:
I don't know why there are sapces but i will update these

Reply:
The spaces are used so that the whole 20 characters are filled. Otherwise, the missing characters will use what was before on the display.
Other status screen messages have better implementations that don’t need this hack. Might be worth looking into it to save some more bytes of progmem